src/into/docker/container.clj

```diff
@@ -79,7 +79,7 @@
               :params {:name  container-name
                        :body {:Image full-name
                               :User  user
-                              :Cmd   ["tail" "-f" "/dev/null"]
+                              :Entrypoint ["tail" "-f" "/dev/null"]
```

Comment:
That works, but just to be safe (and very explicit), I'd suggest to use:

```suggestion
                              :Entrypoint nil
                              :Cmd ["tail" "-f" "/dev/null"]
```

Reply:
Ah, yes, that is probably better.